pkg/environment/operator.go

```diff
@@ -7,3 +7,14 @@ func IsOperator() bool {
 	_, runOperator := os.LookupEnv("RUN_OPERATOR")
 	return runOperator
 }
+
+// IsOperator returns true when RUN_OPERATOR env var is present
+func IsWebhookServer() bool {
```

Comment:
Sorry, but the webhook chnages are quite critical. Please limit all PRs to the bone. It should not introduce any refactoring, nothing that is not mandatory to perform the cert handling change. Only that way we can properly review it. Please open  separate PRs or commits (but PRs would be better) for refactoring.

Reply:
Well this PR introduce reading the env var twice so it's a good moment to pot next to the other, there is not much refactoring that this and it's quite simple.